build-tools/packages/build-cli/src/lib/package.ts

```diff
@@ -0,0 +1,453 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line unicorn/import-style
+import { posix as path } from "path";
+import { Context, readJsonAsync, Logger, Package, MonoRepo } from "@fluidframework/build-tools";
+import { isInternalVersionScheme } from "@fluid-tools/version-tools";
+import { compareDesc } from "date-fns";
+import ncu from "npm-check-updates";
+import * as semver from "semver";
+import { isReleaseGroup, ReleaseGroup, ReleasePackage } from "../releaseGroups";
+
+/**
+ * Returns the name of a package without the scope.
+ *
+ * @param pkgName - The full scoped package name.
+ * @returns The package name without the scope.
+ *
+ * @internal
+ */
+export function getPackageShortName(pkgName: string): string {
+    const split = pkgName.split("/");
+
+    // Handle unscoped packages
+    if (split.length === 1) {
+        return split[0];
+    }
+
+    // Handle scoped packages
+    const name = split.pop();
+    if (name === undefined) {
+        throw new Error(`Error getting the package short name for: ${pkgName}`);
+    }
+
+    return name;
+}
+
+/**
+ * Checks the npm registry for updates for a release group's dependencies.
+ *
+ * @param context - The {@link Context}.
+ * @param releaseGroup - The release group to check.
+ * @param bumpType - The bump type.
+ * @param prerelease - If true, include prerelease versions as eligible to update.
+ * @param depsToUpdate - An array of packages on which dependencies should be checked.
+ * @param log - A {@link Logger}.
+ * @returns An array of packages that had updated dependencies.
+ *
+ * @internal
+ */
+// eslint-disable-next-line max-params
+export async function npmCheckUpdates(
+    context: Context,
+    releaseGroup: ReleaseGroup | ReleasePackage,
+    depsToUpdate: ReleasePackage[] | RegExp[],
+    bumpType: "patch" | "minor" | "current",
+    // eslint-disable-next-line default-param-last
+    prerelease = false,
+    // eslint-disable-next-line default-param-last
+    writeChanges = false,
```

Comment:
`writeChanges` is missing `@param` docs. Also the params list is out-of-order relative to the actual function paramter ordering.